Backpack/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonTest.kt

```diff
@@ -52,16 +58,19 @@ class BpkButtonTest {
 
   @Test
   fun test_enabled_state() {
-    val button = BpkButton(context, BpkButton.Type.Primary).apply {
-      isEnabled = true
-    }
-    val newState = false
-    val expectedBackgroundState = button.disabledBackground()
-    button.isEnabled = newState
+    // android.util.AndroidRuntimeException: Animators may only be run on Looper threads
+    activity.runOnUiThread {
```

Comment:
if (Looper.myLooper() == null) {
      Looper.prepare()
    }

Would that work?